tests/stress_tests/mutex_stress/src/main.c

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <ztest.h>
+
+#define THREAD_COUNT 2
+#define THREAD_STACK_SIZE 1024
+
+struct k_thread thread_array[THREAD_COUNT];
+K_THREAD_STACK_ARRAY_DEFINE(thread_stack, THREAD_COUNT, THREAD_STACK_SIZE);
+
+struct k_thread sem_thread;
+K_THREAD_STACK_DEFINE(sem_thread_stack, THREAD_STACK_SIZE);
+
+struct k_mutex mutex;
+struct k_sem sem;
+struct k_timer timer;
+
+void clean_up(struct k_timer *timer)
+{
+	ARG_UNUSED(timer);
+
+	int i;
+
+	for (i = 0; i < THREAD_COUNT; i++) {
+		k_thread_abort(&thread_array[i]);
+	}
+	k_thread_abort(&sem_thread);
+
+}
+
+void thread_func(void *p1, void *p2, void *p3)
+{
+	int ret, index;
+
+	index = (intptr_t)p1;
+	k_sleep(K_MSEC(100));
```

Comment:
Same here, I think better to put that "100" into a variable. Also please add comment why sleep is necessary here.

Reply:
I added some comments for those sleep operations, thanks.